### ISSUE TITLE
Lack of comments

### ISSUE THREAD

**Issue by TrexSlayer-0** [ArsxnUses/BMI-Calculator#2]
Hello ArsxnUses, This is a fascinating BMI calculator that is very good at measuring my BMI after I eat all the vegan ice cream, but I have a concern regarding the code. There is a lack of comments where I can't tell what the code does, so it would be helpful if you put comments so I can understand it. I appreciate any help you can provide.

**Comment by ArsxnUses** [comment]
Sorry to hear your having issues reading the code, as this program was an assignment for class, I chose to remove the comments to protect myself and others. I try to make my code as understandable as possible however I can see how it's hard to follow, if you would like more help understanding please message me directly and I can explain further.
